### PR TITLE
Adding csv-to-api and rdbms-to-api

### PR DESCRIPTION
**Pull request by torcato**

### CODE REVIEW

File: 1/public/csv-to-api/airport-search.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# yaml-language-server: $schema=https://cdn.raw-labs.com/endpoint-schemaV0.9.json
+raw: 0.9
+endpoint: GET
+metadata:
+  title: Search airport
+  description: "This endpoint has 3 optional arguments:
```
Comment (miguel-branco) [inline 1/public/csv-to-api/airport-search.yml:6]: Is " the right syntax? I thought in YAML there was a | trick.
But I'm not a YAML expert.

File: 1/public/csv-to-api/code.rql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+number_airports_per_country() := {
+    SELECT Country, COUNT(*) AS Number_Airports
+        FROM airports
+        GROUP BY Country
+        ORDER BY Number_Airports DESC
+}
+
+
+// Search airports by country and  city
+search(country: string := "", city: string := "", IATA: string := "") := {
```
Comment (miguel-branco) [inline 1/public/csv-to-api/code.rql:13]: We could make default be "%" then the user can override it.

File: 1/public/rdbms-to-api/airport-by-id.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=https://cdn.raw-labs.com/endpoint-schemaV0.9.json
+raw: 0.9
+endpoint: GET
+metadata:
+  title: Get a single airport by ID in the database
+  description: "Get a single airport by id using the mandatory argument id:
+    
+    rdbms-to-api/airport-by-id?ID=<airport_id>"
```
Comment (miguel-branco) [inline 1/public/rdbms-to-api/airport-by-id.yml:8]: Same comment on "

File: 1/public/rdbms-to-api/airport-by-id.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  public: true
+computeClass: normal
+enabled: true
+tests:
+  - name: single_airport
+    description: get airport by id.
+    arguments:
+      - key: id
+        value: "123"
+
```
Comment (miguel-branco) [inline 1/public/rdbms-to-api/airport-by-id.yml:25]: Empty line.

**Review by miguel-branco — COMMENTED** [review]
I think CSV and RDBMS should have the same exact APIs and code: only the "READ" would change.
My first take was to do what's in here: https://github.com/raw-labs/vscode-templates/tree/main/1-csv
But as @torcato suggested, add in one call to look by ID, since it's indeed a very common operation.

File: 1/public/rdbms-to-api/code.rql [hunk 5]
```diff
@@ -0,0 +1,37 @@
+//1 - Register a new postgres database by going to https://app.raw-labs.com/admin/repositories  
+//   and clicking in "Credentials" -> "PostreSQL" 
+//2 - Read a table using the read_pqsql with the credential name you just regitered.
+//   In this example we named it  "demo-pg-server" and we are reading the table 'airports' in the schema 'public'.
```
Comment (miguel-branco) [inline 1/public/rdbms-to-api/code.rql:4]: These comments are not relevant here.
At most we can explain that READ_PGSQL is using a database of ours.

**Review by glagardere — APPROVED** [review]
Approve for trial

**Pull request merged**